lib/app.js

```diff
@@ -1,17 +1,17 @@
 const express = require('express');
 const cookieParser = require('cookie-parser');
-
-const authenticate = require('./middleware/authenticate');
-
+const cors = require('cors');
 const app = express();
 
 // Built in middleware
 app.use(express.json());
 app.use(cookieParser());
+app.use(cors({ origin: ['http://localhost:5500'], credentials: true }));
 
 // App routes
 
 app.use('/api/v1/users', require('./controllers/users'));
+app.use('/api/v1/items', require('./controllers/items'));
```

Comment:
Since every route in items requires the authenticate middleware, you can put it here at the top level instead of placing it in every route in the controller.